partiql-plan/src/main/resources/partiql_plan.ion

```diff
@@ -41,6 +41,7 @@ ref::{
     cast::{
       input: partiql_value_type,
       target: partiql_value_type,
+      isNullable: bool
     }
```

Comment:
>  I've updated the definition of Ref.Cast to contain isNullable -- at least until we truly represent casts with function calls

Well, we did have them as function calls but that was a pain to actually work with. While the SQL specification may define them like function calls, we don't necessarily have to implement them as such if it's more trouble than its worth.

Why add isNullable here? This is the cast specification, and every cast propagates null. So the nullability of a cast is entirely determined by the input argument, unless I'm missing something related to CAST(.. AS NULL)